core/src/main/java/jenkins/util/java/JavaUtils.java

```diff
@@ -43,19 +43,23 @@ private JavaUtils() {
     /**
      * Check whether the current JVM is running with Java 8 or below
      * @return {@code true} if it is Java 8 or older version
+     * @deprecated because the current version is at least Java 17,
+     * this method is redundant.
      */
+    @Deprecated(since = "TODO", forRemoval = true)
     public static boolean isRunningWithJava8OrBelow() {
-        String javaVersion = getCurrentRuntimeJavaVersion();
-        return javaVersion.startsWith("1.");
+        return false;
     }
 
     /**
      * Check whether the current JVM is running with Java 9 or above.
      * @return {@code true} if it is Java 9 or above
+     * @deprecated because the current version is at least Java 17,
+     * this method is redundant.
      */
+    @Deprecated(since = "TODO", forRemoval = true)
     public static boolean isRunningWithPostJava8() {
```

Comment:
Neither of these methods are used in jenkinsci 

https://github.com/search?q=org%3Ajenkinsci+isRunningWithPostJava8&type=code

https://github.com/search?q=org%3Ajenkinsci%20isRunningWithJava8OrBelow&type=code

Can we just remove them?

Reply:
Thanks for asking the question.  I confirmed with [usage in plugins](https://www.jenkins.io/doc/developer/plugin-development/usage-in-plugins/) that those two methods are unused in Jenkins plugins.  I also searched the cloudbees organization on GitHub and confirmed they are unused there as well.

Removed in ed9b418c499e37435a79845c5220dc81f7589c93